### ISSUE TITLE
kpoof daemon just stop working sometimes

### ISSUE THREAD

**Issue by gerbil** [farmotive/kpoof#19]
I can see random connection issues with kpood daemon on my local kuber cluster, without any reasons..  it just stops working so i need to restart it. 
any ideas how can i troubleshoot this?

**Comment by lypht** [comment]
Are other long-running processes also timing out?  Is the local cluster running on a laptop with any power-saving mode (clam-shell close, for example)?  The daemon doesn't withstand power cycles or similar OS-governed processes.

**Comment by gerbil** [comment]
Nope, it's centos based cluster on vms in my dc; All kubeletes running with connection timeout=0

**Comment by gerbil** [comment]
Weird stuff, sometime in the middle of the night:
E0503 02:16:58.605281    5471 portforward.go:233] lost connection to pod

How can i detect those failures and fix them automagically?

**Comment by lypht** [comment]
By any chance are you doing large pipes to the pod, such as from a mongodump or similar?  If the throughput is greater than the pod can handle, it will close the connection.
